index.html

```diff
@@ -50,28 +50,29 @@ <h3 class="social">LET’S CONNECT</h3>
         </section>
         <section id="work">
```

Comment:
- [ ] @team great work in your popup however the entire work section is supposed to be generated dynamically using javascript not hard coded as you have done, you have already created an array, just using the same data to populate this section, here are examples, eg [example1](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96) or [example2](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)